app/gui/src/dashboard/layouts/CategorySwitcher.tsx

```diff
@@ -32,11 +29,11 @@ import { useEventCallback } from '../hooks/eventCallbackHooks'
 import { useSetCurrentDirectoryId } from '../providers/DriveProvider'
 import { useCloudCategoryList, useLocalCategoryList } from './Drive/Categories/categoriesHooks'
 
-/** Metadata for a categoryModule.categoryType. */
+/** Metadata for a category. */
 interface CategoryMetadata {
   readonly isNested?: boolean
   readonly category: Category
-  readonly icon: string
+  readonly icon: ariaComponents.IconProp & string
```

Comment:
Doesn't string here removes autocomplete?

Reply:
yeah... couldn't get it to work for some reason so i just copied `SvgUseIcon` directly